rules/qc.smk

```diff
@@ -448,7 +448,7 @@ def get_peak_calling_qc(sample):
     # deeptools profile
     assembly = treps.loc[sample, "assembly"]
     # TODO: replace with genomepy checkpoint in the future
-    if assembly.lower() in ["ce10", "dm3", "hg38", "hg19", "mm9", "mm10", "grch38.p13", "grch38"]:
+    if assembly.lower() in ["ce10", "dm3", "hg38", "hg19", "mm9", "mm10", "grch38.p13", "grch38", "danrer11"]:
```

Comment:
what's going on here? It looks like the blacklist genomes, but there's Ensembl builds, and zebrafish?

Reply:
Checking if an assembly has an annotation. We can't do this with checkpoints unfortunately.. The list is definitely incomplete. Maybe genomepy has an api?

Reply:
There will be a unified method of obtaining the annotation URL (if it exists) in the next version. Currently there is none. Get Simon to accept the PR :p

That said, I get your intentions and this would indeed be a good point for a checkpoint.